### PR TITLE
Merge branch 'main' into Aira's-Branch-(￣▽￣)-T-T

### PR DESCRIPTION
**Pull request by TheRochero**

### CODE REVIEW

File: excercise1/AIRA/MyMaths.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+def myMultiplication(a, b):
+    return a * b
+
+def mySubtraction(a, b):
+    return a - b
+
+def myDivision(a, b):
+    return a / b #When you are asked something, you take in the knowledge, you process the knowledge, and then you return with an answer. We can make the laptop do the same by writing return.
+
```
Comment (moi-san) [inline excercise1/AIRA/MyMaths.py:12]: remove the unnecessary spaces and please do camelcase

File: excercise1/AIRA/libimportbackuporiginal.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    print("We are doing Mathematics")
+    a = int(input("Enter your first number for maths"))
+    b = int(input("Enter your second number for maths"))
+
+    print(myMaths.myAddition(a, b))
+    print(myMaths.mySubtraction(a, b))
+    print(myMaths.myMultiplication(a, b))
+    print(myMaths.myDivision(a, b))
+
+
```
Comment (moi-san) [inline excercise1/AIRA/libimportbackuporiginal.py]: remove unnecessary spaces and camelcase please

**Review by moi-san — DISMISSED** [review]

File: excercise1/AIRA/libimportbackuporiginal.py (hunk 2 above)
Comment (Adhyaisthebest) [reply excercise1/AIRA/libimportbackuporiginal.py]: correct

Comment (Adhyaisthebest) [inline excercise1/AIRA/libimportbackuporiginal.py]: remove these spaces make camel case!

**Review by Adhyaisthebest — CHANGES_REQUESTED** [review]

**Review by Adhyaisthebest — APPROVED** [review]

**Comment by Adhyaisthebest** [comment]
camel case

**Comment by Adhyaisthebest** [comment]
camel case

File: excercise1/AIRA/MyMaths.py (hunk 1 above)
Comment (Adhyaisthebest) [reply excercise1/AIRA/MyMaths.py:12]: camel case

**Review by Adhyaisthebest — CHANGES_REQUESTED** [review]

**Review by Adhyaisthebest — APPROVED** [review]

**Pull request merged**